lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -13,6 +13,7 @@ graphitesend
 azure-storage==0.32.0
 # PyRods not in PyPI
 python-ldap==2.4.27
+cloudbridge==0.3.1
```

Comment:
No specific reason, I updated to the current latest version (also see starforge PR [#138](https://github.com/galaxyproject/starforge/pull/138)).